ci/release/bugfix-autorelease-branch-pr-body.md

```diff
@@ -0,0 +1,50 @@
+Auto: Release $BUGFIX_VERSION and merge into $SOURCE_BRANCH
+
+# Release $BUGFIX_VERSION
+
+This PR includes changes needed to perform the $BUGFIX_VERSION release.
+When the PR was created, the bugfix release tag $BUGFIX_VERSION should have
+been created on `$TARGET_BRANCH`, which should also have triggered builds
+for python wheels as well as Windows and Mac binaries.  These binaries
+will be automatically uploaded to the release object at
+https://github.com/$GITHUB_REPOSITORY/releases/tag/$BUGFIX_VERSION.
+
+## If something doesn't look right
+
+1. Decline this PR.  Do not delete the `$TARGET_BRANCH` branch.
+2. Go to the created release object and delete the binaries that have been
+   automatically uploaded there:
+     1. Go to https://github.com/$GITHUB_REPOSITORY/releases/tag/$BUGFIX_VERSION.
+     2. Click "Edit"
+     3. For each binary file, click the "X" button on the right-hand side.
+     4. Click the green "Update Release" button at the bottom of the
+        release page.
+     5. Click the red "Delete" button to delete the release.
+3. PR whatever changes are needed into `$TARGET_BRANCH`.
+   ```shell
+   $ git checkout master
+   $ git pull upstream master
+   $ git checkout $TARGET_BRANCH
+   < make and commit any needed changes >
+   $ git push origin $TARGET_BRANCH
+   ```
+4. Once things look right, tag `$TARGET_BRANCH` with `$BUGFIX_VERSION` and
+   push to `$TARGET_BRANCH`.
+   ```shell
+   $ git checkout $TARGET_BRANCH
+   $ git tag --force $BUGFIX_VERSION
+   $ git push git@github.com:$GITHUB_REPOSITORY.git $TARGET_BRANCH $BUGFIX_VERSION
+   ```
+   Re-tagging and pushing the files to `$TARGET_BRANCH` will cause the release
+   binaries to be rebuilt and re-uploaded to the release object.
+5. Submit a PR from `$GITHUB_REPOSITORY:$TARGET_BRANCH` into `$GITHUB_REPOSITORY:$SOURCE_BRANCH`.
+
+
+## If everything looks OK
+
+1. Approve and merge this PR.
+2. Delete the branch `$TARGET_BRANCH` once the PR is merged.
+3. Take a look at the release object and make sure everything is there:
+   https://github.com/$GITHUB_REPOSITORY/releases/tag/$BUGFIX_VERSION.
+4. Upload python wheels to PyPI.  An issue was created for this,
+   so remember to mark progress there as well.
```

Comment:
Are there any other important steps? I'm thinking about our normal release checklist. In particular, updating natcap website links (though I think they may be automated somewhere else) and running `invest_autotest.py`. That sort of manual testing starts to defeat the purpose though, and maybe it really is overkill for bugfix releases.

Reply:
I've added a couple of notes in 10a7af5e, including a link to the release checklist which I will need to update to account for these bugfix release-related changes.

Ultimately, I'd love to see all of these forms of tests be automated so that they're just a part of our regular workflow.  In the meantime, running `invest-autotest.py` might be overkill for a bugfix release, and having a quicker release cycle should mean that even if we do have a breaking bug in a bugfix release, we can still issue a fix pretty quickly.